crates/sui-json-rpc/src/api.rs

```diff
@@ -479,6 +480,20 @@ pub trait EventReadApi {
     ) -> RpcResult<EventPage>;
 }
 
+#[open_rpc(namespace = "sui", tag = "Threshold BLS APIs")]
+#[rpc(server, client, namespace = "sui")]
+pub trait ThresholdBlsApi {
+    /// Sign an a Randomness object with threshold BLS.
+    #[method(name = "tblsSignRandomnessObject")]
+    async fn tbls_sign_randomness_object(
+        &self,
+        /// The object ID.
+        object_id: ObjectID,
```

Comment:
Do we only care about the ObjectId of the object or do we want to be more specific and have them sign ObjectId + ObjectVersion?

Reply:
yes, just to check it was created